templates/rule-change-proposal.md

```diff
@@ -0,0 +1,19 @@
+**What rule do you want to change?**
+
+
+**Does this change cause the rule to produce more or fewer warnings?**
+
+
+**How will the change be implemented? (New option, new default behavior, etc.)?**
+
+
+**Please provide some example code that this change will affect:**
+
+```js
+<!-- example code here -->
```

Comment:
Should this be a js comment syntax instead, since it's inside a "```js" code block?

```js
/* example code here */
```